src/components/flytegraph/ReactFlow/customNodeComponents.tsx

```diff
@@ -238,10 +247,24 @@ export const ReactFlowCustomTaskNode = ({ data }: any) => {
       </div>
     );
   };
+
+  const renderCacheIcon = (cacheStatus) => {
+    if (cacheStatus === CatalogCacheStatus.CACHE_HIT) {
+      return <CachedOutlined style={cacheIconStyles} />;
```

Comment:
nit: switch could be a better choice from readability perspective

Reply:
rewrote it in switch this morning, but then decided to use ifs 😆

Reply:
updated!